Deprecate AppVeyor CI from OMR

wbh123456 opened the pull request:
Since Azure CI Builds have been running stably for more than 3 months. We can deprecate the old AppVeyor CI build.

Signed-off-by: Bohao(Aaron) Wang <aaronwang0407@gmail.com>

wbh123456 commented:
Removed AppVeyor build configures in this PR. In order to completely deprecate the AppVeyor CI build, we need to remove the AppVeyor Webhook from the OMR repo setting. @fjeremic is opening up a Bugzilla for this.

0xdaryl commented:
There is also a mention of AppVeyor in the GuideForCommitters.md.  Can you replace it with Azure as I don't believe we added any info on that when we added the Azure pipeline?

wbh123456 commented:
> There is also a mention of AppVeyor in the GuideForCommitters.md. Can you replace it with Azure as I don't believe we added any info on that when we added the Azure pipeline?

I have changed that section in the new commit 1ab9b6d. Sorry for the delay.
